crates/web-sys/src/features/gen_RtcRtpCodecParameters.rs

```diff
@@ -15,9 +15,12 @@ impl RtcRtpCodecParameters {
     #[doc = "Construct a new `RtcRtpCodecParameters`."]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `RtcRtpCodecParameters`*"]
-    pub fn new() -> Self {
+    pub fn new(clock_rate: u32, mime_type: &str, payload_type: u8) -> Self {
```

Comment:
You will have to avoid making any breaking changes as mentioned in the last PR: https://github.com/rustwasm/wasm-bindgen/pull/3742#discussion_r1425368919.